src/plugins/visualizations/public/embeddable/visualize_embeddable.tsx

```diff
@@ -477,6 +478,9 @@ export class VisualizeEmbeddable
       this.handler.events$
         .pipe(
           mergeMap(async (event) => {
+            if (isDimensionsEvent(event)) {
```

Comment:
The renderers now dispatch sizing events as well as filter events. That code originally assumed that all events were filter events. Since the legacy visualizations use the same renderers, they were incorrectly treating sizing events as filters, locking up the filtering system. Now, they just ignore non-filter events.

Reply:
Got it, can you add a comment though?

Reply:
Done.